static/templates/base.html

```diff
@@ -74,16 +74,16 @@
         <div class="container">
             <nav class="navbar navbar-expand-lg navbar-dark bg-trezor">
                 <span class="navbar-nav">
-                    <a class="nav-link" href="https://satoshilabs.com/" target="_blank" rel="noopener noreferrer">© 2021 SatoshiLabs</a>
+                    <a class="nav-link" href="https://satoshilabs.com/" target="_blank" rel="noopener noreferrer">© 2017-2022 SatoshiLabs</a>
```

Comment:
You can use `new Date().getFullYear();`

Reply:
Thanks for the tip! I leave that up to Martin. Maybe he wants to set the date from Go even. I am fine with having it statically too.

Reply:
Thank you. I will leave it static, I think it makes more sense to have hardcoded in the source.
I planned to update it with the next release (and I want to sligthly restyle it as well so that it matches more the Suite design) but you were faster.